projects/1.21/assets/productivebees/productivebees/lang/zh_cn.json

```
@@ -0,0 +1,1090 @@
+  "block.productivebees.advanced_maple_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_maple_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_oak_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_oak_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_palm_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_palm_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_redwood_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_redwood_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_river_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_river_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_rosewood_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_rosewood_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_snake_block_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_snake_block_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_spruce_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_spruce_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_umbran_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_umbran_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_warped_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_warped_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_willow_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_willow_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_wisteria_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_wisteria_canvas": "帆布扩容盒",
+  "block.productivebees.advanced_yucca_canvas_beehive": "高级帆布蜂箱",
+  "block.productivebees.expansion_box_yucca_canvas": "帆布扩容盒",
+  "block.productivebees.petrified_honey": "石化蜂蜜",
+  "block.productivebees.white_petrified_honey": "白色石化蜂蜜",
+  "block.productivebees.black_petrified_honey": "黑色石化蜂蜜",
+  "block.productivebees.gray_petrified_honey": "灰色石化蜂蜜",
+  "block.productivebees.light_gray_petrified_honey": "淡灰色石化蜂蜜",
+  "block.productivebees.blue_petrified_honey": "蓝色石化蜂蜜",
+  "block.productivebees.light_blue_petrified_honey": "淡蓝色石化蜂蜜",
+  "block.productivebees.lime_petrified_honey": "黄绿色石化蜂蜜",
+  "block.productivebees.yellow_petrified_honey": "黄色石化蜂蜜",
+  "block.productivebees.orange_petrified_honey": "橙色石化蜂蜜",
+  "block.productivebees.brown_petrified_honey": "棕色石化蜂蜜",
+  "block.productivebees.green_petrified_honey": "绿色石化蜂蜜",
+  "block.productivebees.magenta_petrified_honey": "品红色石化蜂蜜",
+  "block.productivebees.purple_petrified_honey": "紫色石化蜂蜜",
+  "block.productivebees.pink_petrified_honey": "粉红色石化蜂蜜",
+  "block.productivebees.cyan_petrified_honey": "青色石化蜂蜜",
+  "block.productivebees.red_petrified_honey": "红色石化蜂蜜",
+  "block.productivebees.amber": "琥珀块",
+  "entity.minecraft.villager.productivebees.beekeeper": "养蜂人",
+  "entity.productivebees.bee_bee": "蜜蜂",
+  "entity.productivebees.chaos_bee": "混沌蜜蜂",
+  "entity.productivebees.chaotic_bee": "混沌蜜蜂",
```

我一开始居然没发现，记得同步到1.20.1
```suggestion
  "entity.productivebees.chaos_bee": "混沌蜜蜂",
  "entity.productivebees.chaotic_bee": "混沌锭蜜蜂",
```
混沌蜜蜂来自龙之进化，混沌锭蜜蜂来自L2